server/app/controllers/fusor/api/v21/deployments_controller.rb

```diff
@@ -64,6 +64,19 @@ def deploy
       end
     end
 
+    def redeploy
+      begin
```

Comment:
@eriknelson, @jmrodri, I also agree with having a separate method for `redeploy`. On a differenent note, we can clean up some code and move `def deploy` from v2 to v21. I don't think we need to maintain a v2 version for fusor. I think it was there for historic reasons before we switched to v21.